act-rules/act-rules.github.io
# Update rule templates

Update rule templates with the new "acknowledgements", "authors" and "previous_authors" keys.

Resulting from #997 


Closes issue(s):

- (ADD ISSUE NUMBER HERE)

Need for Final Call:
This will not require a Final Call -- not a rule.

---

## Pull Request Etiquette

### **When creating PR:**

- Make sure you requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- Add yourself (and co-authors) as "Assignees" for PR.
- Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)
- Optionally request feedback from anyone in particular by assigning them as "Reviewers".

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also r1eview the proposed Final Call period. In case of disagreement, the longer period wins.
Resulting from #997
